src/stateToHTML.js

```diff
@@ -157,19 +156,19 @@ class MarkupGenerator {
   inlineStyles: StyleMap;
   styleOrder: Array<string>;
 
-  constructor(contentState: ContentState, options: ?Options) {
-    if (options == null) {
-      options = {};
-    }
-    this.contentState = contentState;
-    this.options = options;
-    let [inlineStyles, styleOrder] = combineOrderedStyles(
-      options.inlineStyles,
-      [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER],
-    );
-    this.inlineStyles = inlineStyles;
-    this.styleOrder = styleOrder;
-  }
+  constructor(contentState: ContentState, options: ?Options) {
+    if (options == null) {
+      options = {};
+    }
+    this.contentState = contentState;
+    this.options = options;
+    let [inlineStyles, styleOrder] = combineOrderedStyles(
+      options.inlineStyles,
+      [DEFAULT_STYLE_MAP, DEFAULT_STYLE_ORDER],
+    );
+    this.inlineStyles = inlineStyles;
+    this.styleOrder = styleOrder;
+  }
```

Comment:
What's the difference between the code on the left and the code on the right? If there are no changes can you rebase this PR so they are the same?